requirements/requirements.txt

```diff
@@ -3,7 +3,7 @@ python-dateutil>=2.6, <3.0
 watchdog>=2.1, <3.0
 combo-lock>=0.2.2, <0.3
 
-padacioso>=0.1.2, <0.2.0
+padacioso~=0.2
```

Comment:
is this needed? the min version didnt really change, maybe <0.3.0 is enough

Reply:
I could mark ~=0.1, but it won't be backwards-compatible with Padatious. I don't think we should pin max versions since any breaking changes should be in >=1.0.0

Reply:
~=0.2.0 is pinning a max and min version too... this should be the less strict possible

Reply:
`~=0.2` so it only pins `<1.0.0` for an upper limit since `1.0.0` is where breaking changes may be introduced